release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1100,6 +1100,12 @@ The 1.6.2 version of the Network Observability Operator fixes a compatibility is
 
 In this release, the firmware search path has been updated to copy the contents of the specified path into the path specified in `worker.setFirmwareClassPath` (default: `/var/lib/firmware`). For more information, see xref:../hardware_enablement/kmm-kernel-module-management.adoc#kmm-example-cr_kernel-module-management-operator[Example Module CR].
 
+
+[id="ocp-4-17-kmmo-rhel-els-minimal_{context}"]
+==== Kernel Module Management Operator on rhel-els-minimal
+
+KMM images are now based on rhel-els-minimal container images instead of the rhel-els images. This change results in a greatly reduced image footprint, while still maintaining FIPS compliance.   
```

Comment:
```suggestion
The Kernel Module Management (KMM) Operator images are now based on rhel-els-minimal container images instead of the rhel-els images. This change results in a greatly reduced image footprint, while still maintaining FIPS compliance.   
```
I'm not familiar enough with this documentation set to know, but should "rhel-els-minimal" and "rhel-els" be in monospace font?

Reply:
@lahinson Yes. I checked other sources and that's how it's used.